data/product.d/fedora.conf

```diff
@@ -11,6 +11,7 @@ efi_dir = fedora
 
 [Storage]
 default_scheme = BTRFS
+btrfs_compression = zstd:1
```

Comment:
There are Fedora-based products that don't use Btrfs as a default scheme. Should the compression be disabled for these products?

Reply:
Compression only affects Btrfs filesystems created in Anaconda, so it shouldn't have any effect for Fedora products that choose to use other filesystems.